tests/contrib/futures/test_futures_patch_generated.py

```diff
@@ -27,3 +26,6 @@ def assert_not_module_patched(self, concurrent_futures):
 
     def assert_not_module_double_patched(self, concurrent_futures):
         pass
+
+    def assert_module_implements_get_version(self):
```

Comment:
This method is unimplemented. Is this intentional?

Reply:
Yes, since futures has no version. Otherwise the tests default to the base class case which is testing for get_version and results in a fail